### PR TITLE
boards/opentitan: Update the README

### PR DESCRIPTION
**Pull request by alistair23**
### Pull Request Overview

Update the README to fix some outdated information and point users to
use our commit and steps.

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [X] Ran `make prepush`.

### CODE REVIEW

**Review by phil-levis — CHANGES_REQUESTED** [review]
Please link to the OpenTitan documentation for the specific commit, not the most up to date documentation. Otherwise the OpenTitan documentation is incorrect. 

I recommend that you go through the instructions in a clean VM exactly as they're written to make sure they work as is.

File: boards/opentitan/README.md [hunk 1]
```diff
@@ -91,9 +86,12 @@ pip3 install --user -r python-requirements.txt
 
 LANG="en_US.UTF-8" fusesoc --cores-root . run --flag=fileset_top --target=sim --setup --build lowrisc:dv:chip_verilator_sim
 ```
+
 ### Build Boot Rom/OTP Image
+
 Build only the targets we care about.
 ```shell
+./meson_init.sh
```
[the remark this answers is not in the document]
Comment (phil-levis) [reply boards/opentitan/README.md:94]: Yes -- this was the source of about half of my confusion; these instructions are for the specific prior commit, but it links to the OT documentation for the current commit, which uses Bazel.

[the remark this answers is not in the document]
Comment (alistair23) [reply boards/opentitan/README.md:94]: Yep, already on it: https://github.com/tock/tock/pull/3056

The problem is there are ePMP issues with the newer versions so this is a stop gap until we can update

File: boards/opentitan/README.md [hunk 2]
```diff
@@ -20,19 +20,16 @@ Programming
...
-For more information you can follow the
-[OpenTitan development flow](https://docs.opentitan.org/doc/ug/getting_started_fpga/index.html#testing-the-demo-design)
-to flash the image.
-
-First setup the development board using the steps here:
-https://docs.opentitan.org/doc/ug/getting_started_fpga/index.html.
-You need to make sure the boot ROM is working and that your machine can
-communicate with the OpenTitan ROM. You will need to use the `PROG` USB
-port on the board for this.
+In general the OpenTitan project has
```
Comment (phil-levis) [inline boards/opentitan/README.md:27]: Please cut this link -- better to just point people at what the right thing to do is than say "Here's the documentation, actually that's the wrong documentation instead look at this other documentation."

**Review by phil-levis — CHANGES_REQUESTED** [review]

**Comment by hudson-ayers** [comment]
I tried to follow these instructions for the newer commit that we are trying to upgrade OT to, and ran into two issues:

1. `$ ci/scripts/build-chip-verilator.sh earlgrey` hangs with no output after running the fusesoc command (I waited for ~40 minutes, is there a chance I could go that long with no output?)
2. `ninja -C build-out all` fails for a bunch of targets, citing missing files in the OT repository -- maybe I missed a submodule I was supposed to fetch or something?. Rerunning with `ninja -k 0 -C build-out all` gets further and builds some targets, but still seems to fail for some of the targets we need.

This may not be the right place for these complaints, just wanted to note that I tried to follow these instructions and failed. I wanted to spend some time debugging the PMP issues for the new bitstream but without verilator I don't have a practical way to do so

File: boards/opentitan/README.md [hunk 3]
```diff
@@ -20,19 +20,16 @@ Programming
...
+[getting started documentation](https://docs.opentitan.org/doc/ug/getting_started_fpga/index.html)
+avaliable.
+Unfortunately this generally is out of sync with the Tock setup. For instructions
+that match the OpenTitan version Tock supports you will need to read the
+[raw docs via git](https://github.com/lowRISC/opentitan/tree/199d45626f8a7ae2aef5d9ff73793bf9a4233711/doc/ug).
```
[the remark this answers is not in the document]
Comment (alistair23) [reply boards/opentitan/README.md:32]: Fixed

**Comment by alistair23** [comment]
These instructions won't work for the newer commits, as the build system has swapped to Bazel. Have a look at https://github.com/tock/tock/pull/3056 instead.

You should see some signs of life in around 5 minutes on Verilaotr, so 40 minutes seems too long

**Comment by phil-levis** [comment]
Alistair, have you tried running through these instructions on a clean system with the named commit?

**Comment by alistair23** [comment]
Yep!

**Review by bradjc — APPROVED** [review]

**Comment by alistair23** [comment]
This should be ready to go, then I can rebase https://github.com/tock/tock/pull/3056 on top of it which should help for the newwer bazel builds

**Pull request merged**